data_structures/tuples.vale

```diff
@@ -1,55 +1,10 @@
+import stdlib.*;
+
 fn main() export {
   t1 = [23, 108, 37, 42, 49];    // tuple with same element type int
   println(t1.1);                 // => 108
 
   t2 = [5, true, "V"];           // different element types
   println("Babylon " + t2.0);    // => Babylon 5
   println("Saturn " + t2[2]);    // => Saturn V
-
```

Comment:
All these removals are because this was actually an array under the hood. Earlier versions tried to make arrays and "uniform" tuples interchangeable, but I took out that interchangeability in this release.